cishouseholds/edit.py

```diff
@@ -792,3 +793,29 @@ def edit_to_sum_or_max_value(
         .cast("integer"),
     )
     return df
+
+
+def add_work_from_home_identifier(
+    df: DataFrame,
+    columns_to_check_in: List[str],
+    regex_pattern: str = "(W(K|ORK.*?) F(ROM?) H(OME?))|(WFH)",
+    pattern_result_column: str = "is_wfh",
+):
+    """
+    Applies the user provided RegEx pattern to the list of columns. If a value in any of the
+    columns matches the RegEx pattern then `pattern_result_column` column will have the corresponding
+    value set to integer: 1 (True), 0 (False) otherwise.
+
+    Parameters:
+    -----------
+    df: The input dataframe to process
```

Comment:
With [our docstring format](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) the description of the parameters goes on an indented newline

Don't worry about specifying type, as the type hinting above documents this nicely